presto-benchmark/src/main/java/com/facebook/presto/benchmark/AbstractOperatorBenchmark.java

```diff
@@ -215,16 +216,18 @@ private Split getLocalQuerySplit(Session session, TableHandle handle)
     protected final OperatorFactory createHashProjectOperator(int operatorId, PlanNodeId planNodeId, List<Type> types)
     {
         ImmutableMap.Builder<Symbol, Type> symbolTypes = ImmutableMap.builder();
-        ImmutableMap.Builder<Symbol, Integer> symbolToInputMapping = ImmutableMap.builder();
+        ImmutableList.Builder<VariableReferenceExpression> variables = ImmutableList.builder();
+        ImmutableMap.Builder<VariableReferenceExpression, Integer> variableToInputMapping = ImmutableMap.builder();
         ImmutableList.Builder<PageProjection> projections = ImmutableList.builder();
         for (int channel = 0; channel < types.size(); channel++) {
-            Symbol symbol = new Symbol("h" + channel);
-            symbolTypes.put(symbol, types.get(channel));
-            symbolToInputMapping.put(symbol, channel);
+            VariableReferenceExpression variable = new VariableReferenceExpression("h" + channel, types.get(channel));
+            symbolTypes.put(new Symbol(variable.getName()), types.get(channel));
+            variables.add(variable);
+            variableToInputMapping.put(variable, channel);
             projections.add(new InputPageProjection(channel, types.get(channel)));
         }
 
-        Optional<Expression> hashExpression = HashGenerationOptimizer.getHashExpression(ImmutableList.copyOf(symbolTypes.build().keySet()));
+        Optional<Expression> hashExpression = HashGenerationOptimizer.getHashExpression(ImmutableList.copyOf(variables.build()));
```

Comment:
nit: Do we need to do a `ImmutableList.copy` over `variables.build()`? (Since it already generates an `ImmutableList`

Reply:
Probably not, didn't pay attention in refactoring...